.github/workflows/ci.yml

```diff
@@ -88,3 +88,42 @@ jobs:
       - "lint"
       - "build"
       - "pylint"
+  publish_gh:
+    name: "Publish to GitHub"
+    runs-on: "ubuntu-20.04"
+    if: "startsWith(github.ref, 'refs/tags/v')"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Set up Python"
+        uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+      - name: "Install Python Packages for Linting"
+        run: "pip install invoke poetry toml"
+      - name: "Set env"
+        run: "echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:10})"
```

Comment:
is this needed because of the release action used?

Reply:
It's how I have found from researching to get the equivilant tagged version. We are looking to get just the v1.x.x part. Where as the GITHUB_REF comes in with /ref/tags/v1.xx.